src/Components/Mission.js

```diff
@@ -68,28 +70,52 @@ function Mission(props) {
     return (
     <div key={subMission.id} className={subMission.__typename}>         
         <ul>
-          {subMission.name}
+          <h2>Sub-Mission:</h2>
+          <h5>{subMission.name}</h5>
+          
         </ul>
       </div>
     )
   }
 
   return (
     <div className="MissonOverview"> 
-      <h1>{title}</h1>
-      <h2>{description}</h2>
-      <div className="row">        
-        <div className="column">    
+      <div className="background"> 
+        <div className="missionTitle"></div>
+          <div className="missiontext">
+            <h1>{title}</h1>
+            <h2>{description}</h2>
+          </div>
+        <div className="row">        
+          <div className="column">  
+            <div className="taskSpace">
+              <ul>{displayMissions(loading, error, data)}</ul>                         
+            </div>  
+          </div>
+          
+            <div className="preview">
+              <MTaskOverview task={focusedTask}/>
+            </div>
+          
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export default Mission;
+
+/*
+<div className="row">        
```

Comment:
Might be good to get rid of this commented code, unless it is being saved for some reason.